linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/IncOptimizer.scala

```diff
@@ -749,22 +730,40 @@ final class IncOptimizer private[optimizer] (config: CommonPhaseConfig, collOps:
     def removeInstantiatedSubclass(x: Class): Unit =
       _instantiatedSubclasses -= x
 
-    /** PROCESS PASS ONLY. Concurrency safe except with [[ancestors_=]] */
+    /** PROCESS PASS ONLY. Concurrency safe except with [[updateWith]] */
     def ancestors: List[ClassName] =
       _ancestors
 
+    /** PROCESS PASS ONLY. Concurrency safe except with [[updateWith]]. */
+    def registerAskAncestors(asker: MethodImpl): Unit =
+      ancestorsAskers.put(asker, ())
```

Comment:
Is there any reason that this method moved? It seems to create unnecessary diff.